rst_files/orth_proj.rst

```diff
@@ -946,11 +940,11 @@ basis by QR decomposition and project once more.
 
     Py3 = Q * Q' * y
```

Comment:
Sorry, I know you didn't make the change but in this block it has `Q = Matrix(Q)` but then has algebra which would make the conversion unnecessary.  We should keep them in the returned type as long as possible.  See addition to styleguide.

Reply:
Yeah, when I stuck those in (to avoid issues from `Q'` not being a Matrix) it was often overkill. Definitely wasn't going for style.

Reply:
```
Q, R = qr(X)
a = Q * Q' * y
Q = Matrix(Q)
b = Q * Q' * y
a ≈ b
false
a
3-element Array{Float64,1}:
  1.0000000000000004
  2.999999999999999 
 -2.9999999999999996
b
3-element Array{Float64,1}:
 -0.5652173913043473
  3.2608695652173907
 -2.2173913043478253
```
`Matrix` is necessary here.